appveyor.yml

```diff
@@ -1,40 +1,40 @@
 build: false
-shallow_clone: true
+shallow_clone: false
+skip_branch_with_pr: true
 environment:
 
   global:
-    # SDK v7.0 MSVC Express 2008's SetEnv.cmd script will fail if the
-    # /E:ON and /V:ON options are not enabled in the batch script intepreter
-    # See: http://stackoverflow.com/a/13751649/163740
-    CMD_IN_ENV: "cmd /E:ON /V:ON /C .\\appveyor\\run_with_env.cmd"
+    PYTHONUNBUFFERED: "1"
+    INSTALL_EDM_VERSION: "1.9.1"
```

Comment:
Is it worth adding a comment to the .yml file on why we're not using 1.10?

Reply:
BTW, Why not 1.9.2?

Reply:
added comment and changed edm version to `1.9.2`